lib/matplotlib/tests/test_legend.py

```diff
@@ -14,6 +14,36 @@
 import matplotlib.transforms as mtransforms
 import matplotlib.collections as mcollections
 from matplotlib.legend_handler import HandlerTuple
+import inspect
+
+
+# test that docstrigs are the same
+def get_docstring_section(func, section):
```

Comment:
👍 I like this a lot!

Reply:
Yes, its saved me already, though the cutting and pasting is getting old...